spec/Section 6 -- Execution.md

```diff
@@ -337,6 +337,112 @@ serial):
   selection set.
 - Return an unordered map containing {data} and {errors}.
 
+### Field Collection
```

Comment:
I think the heading hierarchy might be wrong after this change. looking here: https://deploy-preview-1077--graphql-spec-draft.netlify.app/draft/#sec-Executing-a-Field-Plan

Field Collection is now a child of "Executing the Root Selection Set". Did you intend for it to be a sibling of "Executing a Field Plan"?

Reply:
Hmmm. I made it a child of executing the root selection set because that’s when it’s first called.

Although I see why we might want to raise it a level, as it’s also called elsewhere.